internal/version/version.go

```diff
@@ -0,0 +1,23 @@
+package version
```

Comment:
nit: add license header.

Reply:
[SPDX](https://spdx.dev/) license headers are shorter than the standard ones we use (`// SPDX-License-Identifier: Apache-2.0` instead of the multi-line one). 

In any case, this is a discussion that doesn't belong to this PR, so please continue to use the same license header we're using in other files.